include/psa/crypto_values.h

```diff
@@ -1129,6 +1129,23 @@
  */
 #define PSA_ALG_OFB                             ((psa_algorithm_t)0x04c01200)
 
+/** The CCM* cipher mode without authentication.
+ *
+ * This is CCM* as specified in IEEE 802.15.4 §7, with a tag length of 0.
+ * For CCM* with a nonzero tag length, use the AEAD algorithm #PSA_ALG_CCM.
+ *
+ * The underlying block cipher is determined by the key type.
+ *
+ * CCM* requires knowledge of the plaintext length before producing output.
+ * As a consequence, it can only be used with the single-part cipher interface
+ * (psa_cipher_encrypt() and psa_cipher_decrypt()), not with the multi-part
+ * cipher interface.
```

Comment:
That's not convenient: this only allows a maximum-length, random IV.

Let's add a `psa_cipher_set_length()`, similar to `psa_aead_set_lengths()`, only necessary for CCM*.